### DISCUSSION TITLE
No web GUI on port 8091

### DISCUSSION THREAD

**Discussion by fjuppe** [zwave-js/zwave-js-ui#3291] [Q&A]
I am a newbie in this field but I have tried to install on Linux Rasperry Deb 11 (Rpi 4B+) the functionality for Z-wave and Domoticz. Domoticz is fine and connected to port 1883. But I can not get the Web GUI on port 8091 to work from my Windows based laptop. Running no Desktop on RPi. I need help to get this fixed so I can connect my Z-Wave divices through the new Aeontec gen 7 Z-Wave USB stick. Any help appreciated but please be detailed with code. Or, you might log in to my RPi, I have Dyn DNS for this.
Thanks in advance!

**Comment by robertsLando** [comment]
Maybe you have hit: https://github.com/zwave-js/zwave-js-ui/issues/3267

**Reply by fjuppe** [reply]
Maybe, I dont know the command to check version. In browser I just put 192.168.0.28:8091 and I get "Connection refused"
How do I downgrade in case I have the latest version?
Thanks for your fast reply !

**Reply by robertsLando** [reply]
Check the url you have entered and correct it by removing the path like suggested in the issue

**Reply by fjuppe** [reply]
I've tried several URL's but still "connection refused"...

**Reply by robertsLando** [reply]
Got some logs? Could you also place here a screenshot of your web page?

**Reply by fjuppe** [reply]
Edge and Chrome just refuse to connect with error message "ERR_CONNECTION_REFUSED" with or without any path after port number
Where could I look for logs for this issue in the RPi? Could it be a network issue and not a Zwave_JS_UI issue?
Remember, I am a beginner....:_))

**Comment by fjuppe** [comment]
Solved!

**Reply by robertsLando** [reply]
What was the issue at the end?

**Comment by fjuppe** [comment]
I installed in a new container with Snap instead of Docker. There were very clear instructions in forum how to install with Snap.But now I dont know how to unistall the Docker environment…?Med vänlig hälsning/Best regardsTorbjörn Hall26 sep. 2023 kl. 14:13 skrev Daniel Lando ***@***.***>:﻿
What was the issue at the end?

**Reply by robertsLando** [reply]
I dunno if you are using a docker-compose file simply go to the docker compose directory and run:

```
docker-compose down
```

Otherwise f zui is the only container running you can use:

```
# kill running container
docker kill $(docker ps -aq) 
# remove containers
docker rm $(docker ps -aq) 
```

Be aware that those commands will stop and remove ALL your running contianers

**Comment by fjuppe** [comment]
OK. I will try.Thanks a lot for your support!Med vänlig hälsning/Best regardsTorbjörn Hall26 sep. 2023 kl. 15:17 skrev Daniel Lando ***@***.***>:﻿
I dunno if you are using a docker-compose file simply go to the docker compose directory and run:
docker-compose down

Otherwise f zui is the only container running you can use:
# kill running container
docker kill $(docker ps -aq) 
# remove containers
docker rm $(docker ps -aq) 

Be aware that those commands will stop and remove ALL your running contianers
